meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -1,2 +1,5 @@
 # LS1021atwr supports usb-audio & simple soundcard, alsa utils are required for playback/record operations. Also we need to support vfat.
 MACHINE_FEATURES_append_ls1021atwr = " alsa vfat"
+
+# LS1021atwr install rcw into cached-binaries.
```

Comment:
This is wrong. The point I made was that it *is not* installed into cached-binaries, so calling it prebuilt binaries was misleading.